File: src/main/java/no/stunor/origo/batch/model/Eventor.java
package no.stunor.origo.batch.model;

import com.google.cloud.spring.data.firestore.Document;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Document(collectionName = "eventors")
public class Eventor {
    private String eventorId;
    private String name;
    private String federation;
    private String baseUrl;
    private String apiKey;
}

File: src/main/kotlin/no/stunor/origo/batch/model/Eventor.kt
package no.stunor.origo.batch.model

import org.springframework.data.annotation.Id

data class Eventor (
        @Id
        var eventorId: String = "",
        var name: String = "",
        var federation: String = "",
        var baseUrl: String = "",
        var apiKey: String = ""
)